modules/golang/manifests/init.pp

```diff
@@ -7,11 +7,11 @@
   include govuk_ppa
 
   class { 'goenv':
-    global_version => '1.9.1',
+    global_version => '1.15.8',
```

Comment:
Will setting the global version to the latest have any impact on jobs etc where we're not on the latest Go?

Reply:
Yes, if those repositories don't have a `.go-version` file set. I think all repos should explicitly set a go version that they use. I looked through the govuk repos that use go, and found all set a `.go-version` file.

If there is an app that doesn't have this version installed, the reason should be obvious, they'll get an error like `goenv: version '1.15.8' is not installed`.

Reply:
Fair enough, sounds like a reasonable approach. For the recent work installing Terraform we opted to keep the default to what all the projects use and only set the newer version explicitly in the one project which uses it. I think here though each project is using a different version so probably better to let the user know that it fails telling them explicitly which version isn't installed 👍